Scripts for Issue #3

Lischip opened the pull request:
Included a brief explanation on Dask DataFrames in load_multiple_sav_Files.py

Lischip committed:
Scripts for Issue #3

Included a brief explanation on Dask DataFrames in load_multiple_sav_Files.py

harcel commented on code_examples_Py/get_newest_sav_from_dir.py:
Yay, als mezelf in plaats van als DCC! 
Waarom staat de code om dit te fiksen in een namespace-if-block? Kan me voorstellen dat een functie om te doen nuttig is, maar als toch alleen dit gebeurt als dit de __main__ is, dan kan het ook zonder die if....
Waarom willen we alleen weten wat de nieuw_ste_ is, en niet alle nieuwe die nog niet geprocessed zijn (door bijvoorbeeld een lijst bij te houden in een .onzichtbarelijstfile) zodat je weet welke nog moeten?

harcel commented on code_examples_Py/get_sav_files_from_dir.py:
Waarom in de __name__ == "__main__"?

harcel commented on code_examples_Py/load_multiple_sav_files.py:
Voor mensen die dask willen gebruiken: prima.  Ik denk dat veel mensen graag bij pandas blijven, en dat is in veel gevallen ook best prima denk ik. Niet alle files zijn groot, tenslotte. Zullen we er daarvor ook functies bij maken?

Lischip replied:
Bedoel je dat we liever alles aanbieden als functie(s) die je kan kopiëren en aanroepen?

harcel commented on code_examples_Py/loading_saving_parquet.py:
De code in het main block hoeft nooit allemaal te worden uitgevoerd. Het is voor de beginners wellicht niet zo duidelijk dat er een keuze te maken is (en dat dat dan in een name==main block staat maakt het voor beginners denk ik ook niet duidelijker. Is dit code die wij gaan runnen of een onderzoeker?

Lischip committed:
I processed the feedback, see description

get_newest_sav_file zoekt de nieuwste sav file per dir. Hij houdt dus niet bij wat al is "gevonden"/verwerkt, dat is gewoon niet de functie van dat script. Ik kan wel nog een script daarvoor toevoegen aan zowel de Py an R examples. Dat script geeft nu wel een warning als er een nieuwe nieuwste file is.

Instructions were unclear, created load_single_sav_file.py and added comments to load_multiple_sav_files.py.
